rescheduler/rescheduler.go

```diff
@@ -77,8 +88,247 @@ func main() {
 					glog.Errorf("Failed to list unscheduled pods: %v", err)
 					continue
 				}
-				glog.Infof("%+v", allUnschedulablePods)
+
+				criticalPods := filterCriticalPods(allUnschedulablePods)
```

Comment:
@mwielgus I assume you mean "filterCriticalPods() should exclude pods in podsBeingProcessed"? If so, I agree -- good catch.